atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -223,7 +233,8 @@ def run(self):
             "base-image-name": base_image_name,
             "image-id": self.workflow.builder.image_id or '',
             "digests": json.dumps(self.get_pullspecs(self.get_digests())),
-            "plugins-metadata": json.dumps(self.get_plugin_metadata())
+            "plugins-metadata": json.dumps(self.get_plugin_metadata()),
+            "filesystem": json.dumps(self.get_filesystem_metadata()),
```

Comment:
It would be great to add a unit test for ensuring this annotation is set properly.